pmml/step1_prepare/README.md

```
@@ -2,24 +2,24 @@
 
 ## `chapter_count`
 
-![](./step1_4_chapter_count_scatter.png)
+![](./step1_4_chapter_count_scatter_with_regression.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)